platform/app/public/config/kheops.js

```diff
@@ -136,7 +136,7 @@ window.config = {
         supportsReject: true,
         supportsStow: true,
         imageRendering: 'wadors',
-        thumbnailRendering: 'wadors',
+        thumbnailRendering: 'thumbnail',
```

Comment:
Is `thumbnail` the best value here, or should we use something else like `server`? I'm not really sure; maybe it is good.

Reply:
thumbnail is the name of the API - I'd be fine with thumbnailRS to indicate restful service thumbnail?

Reply:
thumbnail is fine